api/authorization-api-1_0.md

```diff
@@ -383,7 +418,8 @@ X-Request-ID: bfe9eb29-ab87-4ca3-be83-a1d5d8305716
 
 {
   "subject": {
-    "id": "atul@sgnl.ai",
+    "format": "email",
```

Comment:
I would suggest using `account`, `iss_sub` or `opaque` in examples rather than `email`.

`email` is a problematic subject identifier because canonicalization algorithms for email addressees aren't that well defined. In some edge cases, it's barely possible to canonicalize email addresses, as a bright example, both user@gmail.com and user+label1@gmail.com "represent" the same email address.